README.md

```diff
@@ -54,17 +77,41 @@ Table of Contents
 
 `syntax: ex, err = expr.new(rule)`
 
-Create a expression object which can be evaluated later.
+Create an expression object which can be evaluated later.
+
+The syntax of rule is an array table of nodes.
+
+The first node can be an expression or a logical operator.
+The remain nodes can be an expression or another array of nodes which contain its logical operator and expressions.
 
-The syntax of rule is an array table of expressions.
-Each expression is an array table which has three elements:
+Each expression is an array table which has three or four elements:
 ```lua
 {
     {"var name (aka. left value)", "optional '!' operator", "operator", "const value (aka. right value)"},
     ...
 }
 ```
 
+Logical operator can be one of
+* OR
```

Comment:
Do we support lowercase expressions, such as'or','and'?

Reply:
No. I think we can implement the uppercase only first, as the uppercase is easier to see. Ignore cases may cause the expressions to mix different cases.